app/controllers/api/v0/ping_controller.rb

```diff
@@ -1,5 +1,11 @@
 class Api::V0::PingController < ApiController
 
+  swagger_controller :ping, 'Ping'
+
+  swagger_api :index do
+    summary 'Responds with "Pong"Returns "Pong" to check the api is alive"'
```

Comment:
Run the documentation endpoint and you can test the API directly and see the output there